sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisResult.java

```diff
@@ -6,86 +6,85 @@
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Stopwatch;
+import com.powsybl.commons.json.JsonUtil;
+import org.jgrapht.alg.util.Triple;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.io.Writer;
 import java.util.*;
-import java.util.stream.Collectors;
+import java.util.concurrent.TimeUnit;
 
 /**
  * Sensitivity analysis result
  *
  * <p>
- *     Mainly composed of the lists of sensitivity values in N, and optionally in N-1
+ *     Composed of a list of sensitivity values in pre-contingency and post-contingency states.
  * </p>
  *
  * A single sensitivity analysis should return, besides its status and some stats on the
- * analysis itself, all the sensitivity values for each factor (combination of a monitoredBranch and a specific
- * equipment or group of equipments). The HADES2 sensitivity provider used with Powsybl offers the
- * possibility to calculate the sensitivity on a set of contingencies besides the N state.
- * The analysis is launched only once, but the solver itself
- * modifies the matrix for each state of the network to output a full set of results.
- * In the sensitivity API, it has been allowed to provide a list of contingencies as an optional input,
+ * analysis itself, all the sensitivity values for each factor (combination of a monitored equipment or bus and a specific
+ * equipment or group of equipments). The chosen sensitivity provider offers the possibility to calculate the sensitivity
+ * on a set of contingencies besides the pre-contingency state.
+ * Note that the analysis is launched only once, but the solver itself modifies the matrix for each state of the network
+ * to output a full set of results. In the sensitivity API, it has been allowed to provide a list of contingencies as an input,
  * which then triggers such a sensitivity analysis.
  * The full set of results consists of :
- *  - the list of sensitivity values in N
- *  - the lists of sensitivity values for each N-1 situation
+ *  - the list of sensitivity values in pre-contingency and post-contingency states.
  *  - some metadata (status, stats, logs)
  *
  * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  * @see SensitivityValue
  */
 public class SensitivityAnalysisResult {
 
-    private static final String VALUE_NOT_FOUND = "Sensitivity value not found for function %s and variable %s.";
-    private static final String VALUE_NOT_FOUND_CONTINGENCY = "Sensitivity value not found for function %s and variable %s at contingencyId %s.";
+    private static final Logger LOGGER = LoggerFactory.getLogger(SensitivityAnalysisResult.class);
 
-    @JsonProperty("ok")
     private final boolean ok;
```

Comment:
a simple ok status, does not make any sens anymore (there is one computation per connected componant). Furthermore, we already know about a computation issue when there is a nan value to the output.